new-log-viewer/src/services/LogFileManager/index.ts

```diff
@@ -226,39 +254,42 @@ class LogFileManager {
      * Gets the data that corresponds to the cursor.
      *
      * @param cursor
-     * @return Log event numbers for:
+     * @return Index for:
```

Comment:
I prefer to specify the index is a
```suggestion
     * @return Log event index for:
```

Reply:
sg. I guess grammatically it should be 
```suggestion
     * @return Indices for:
```